lib/CleantalkSP/Common/Scanner/HeuristicAnalyser/Modules/Entropy.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace CleantalkSP\Common\Scanner\HeuristicAnalyser\Modules;
+
+use CleantalkSP\Common\Scanner\HeuristicAnalyser\Vendors\TiktokenPhp\src\Encoder;
+
+class Entropy
+{
+    /**
+     * @var array|null
+     */
+    private $verdict;
+
+    /**
+     * Flag - is need to check variables separately
+     * @var bool
+     */
+    private $is_file_suspicious;
+
+    /**
+     * @param string $path
+     */
+    public function __construct($path)
```

Comment:
It's like mixing responsibilities. The class does not need to know anything about the language version, this is the client's task.

Reply:
Agree but didn't found any way to implement this in the client code. Please, suggest the code example.